epysurv/simulation/point_source.py

```diff
@@ -3,82 +3,78 @@
 from epysurv.simulation.utils import add_date_time_index_to_frame, r_list_to_frame
+from rpy2 import robjects
 
 surveillance = rpackages.importr("surveillance")
 base = rpackages.importr("base")
 
 
 @dataclass
 class PointSource(BaseSimulation):
-    """Simulation of epidemics which were introduced by point sources.
+    r"""Simulation of epidemics which were introduced by point sources.
 
     The basis of this programme is a combination of a Hidden Markov Model
     (to get random timepoints for outbreaks) and a simple model
-    (compare :class:`epysurv.simulation.seasonal_noise`) to simulate the baseline.
+    (compare :class:`epysurv.simulation.SeasonalNoise`) to simulate the baseline.
 
-    Attributes
+    Parameters
     ----------
     amplitude
-        amplitude (range of sinus), default = 1.
+        amplitude (range of sinus).
     alpha
-        parameter to move along the y-axis (negative values not allowed) with alpha >= amplitude, default = 1.
-    beta
-        regression coefficient, default = 0.
+        parameter to move along the y-axis (negative values are not allowed) with `alpha` >= `amplitude`.
     frequency
-        factor to determine the oscillation-frequency, default = 1.
+        factor to determine the oscillation-frequency.
     p
-       probability to get a new outbreak at time i if there was one at time i-1, default 0.99.
-    phi
-       factor to create seasonal moves (moves the curve along the x-axis), default = 0.
+       probability to get a new outbreak at time :math:`i` if there was one at time :math:`i-1`.
     r
-        probability to get no new outbreak at time i if there was none at time i-1, default 0.01.
+        probability to get no new outbreak at time :math:`i` if there was none at time :math:`i-1`.
+    seasonal_moves
+       seasonal moves (moves the curve along the x-axis).
+
     seed
         a seed for the random number generation
+    trend_parameter
+        trend parameter that controls the influence of the current week on :math:`\mu`.
 
     References
     ----------
-    http://surveillance.r-forge.r-project.org/
+        http://surveillance.r-forge.r-project.org/
     """
 
     alpha: float = 1.0
     amplitude: float = 1.0
-    beta: float = 0.0
     frequency: int = 1
     p: float = 0.99
-    phi: int = 0
     r: float = 0.01
+    seasonal_moves: int = 0
     seed: Optional[int] = None
+    trend_parameter: float = 0.0
 
     def simulate(
         self,
         length: int,
         state_weight: float = 0,
         state: Optional[Sequence[int]] = None,
     ) -> pd.DataFrame:
-        """
-        Simulate outbreak.
+        """Simulate outbreaks.
 
         Parameters
         ----------
         length
-            number of weeks to model, default 100. length is ignored if state is given. In this case the length of state
-            is used.
+            number of weeks to model. ``length`` is ignored if ``state`` is given. In this case the length of
```

Comment:
Let's make every docstring a proper sentence that starts with an uppercase letter and ends with a period.